extension/src/prototypes/ui-components/PropertyParameterItem.tsx

```
@@ -240,7 +254,45 @@ const Property: FC<{
     [values, attrVal, isPrimitive],
   );
 
-  return isPrimitive ? (
+  const displayedValues = useMemo(
+    () =>
```

Biome (1.9.4)

[error] Shouldn't redeclare 'displayedValues'. Consider to delete it or rename it.

'displayedValues' is defined here:


(lint/suspicious/noRedeclare)